src/include/parser/statement_drop.h

```diff
@@ -32,20 +33,29 @@ struct DropStatement : SQLStatement {
   };
 
   DropStatement(EntityType type)
-      : SQLStatement(STATEMENT_TYPE_DROP),
-        type(type),
-        name(NULL),
-        table_name(NULL),
-        missing(false) {}
+      : SQLStatement(STATEMENT_TYPE_DROP), type(type), missing(false) {}
 
   virtual ~DropStatement() {
-    free (name);
-    free (table_name);
+    free(database_name);
+    free(index_name);
+    free(prep_stmt);
+    delete table_name;
+  }
+
+  inline std::string GetTableName() { return std::string(table_name->name); }
```

Comment:
The value of table_name is always initialized with parser, if we check null value, the more interesting question is what value do we return.. empty string / default table name?